test/sqllogictest/string.slt

```diff
@@ -639,3 +639,52 @@ SELECT split_part('one', 'two')
 
 query error arguments cannot be implicitly cast to any implementation's parameters; try providing explicit casts
 SELECT split_part(1, 2, 3)
+
+### lpad ###
+query T
+SELECT lpad('str', 42, 'pad')
+----
+padpadpadpadpadpadpadpadpadpadpadpadpadstr
+
+query T
+SELECT lpad('str', 4, 'pad')
+----
+pstr
+
+query T
+SELECT lpad('str', 5, 'pad')
+----
+pastr
+
+query T
+SELECT REPLACE(lpad('str', 5), ' ', '_')
+----
+__str
+
+query T
+SELECT lpad('str', 7, 'pad')
+----
+padpstr
+
+query T
+SELECT lpad('str', 3, 'pad')
+----
+str
+
+query T
+SELECT lpad('str', 2, 'pad')
+----
+st
+
+query T
+SELECT lpad('str', 1, 'pad')
+----
+s
+
+statement error
+SELECT lpad('str', -1, 'pad')
+
+query T
+SELECT lpad('str', 0, 'pad')
+----
+(empty)
```

Comment:
Could you add tests for what happens:
* when one or some combination of the arguments is null?
* with non-ascii characters in the first and/or third arguments (to test that we are always using character length instead of byte length)
* with the empty string in the first and/or third arguments

Reply:
Could you also add a test for what happens when the second argument is NULL?